scripts/st2bootstrap-deb.sh

```diff
@@ -452,14 +452,15 @@ install_st2_dependencies() {
 }
 
 install_mongodb() {
-  # Add key and repo for the latest stable MongoDB (3.4)
-  # TODO: Install MongoDB 4.0 on Bionic
-  if [[ "$SUBTYPE" == 'bionic' ]]; then
+  # Add key and repo for the latest stable MongoDB (3.6 or 4.0)
+  # Install MongoDB 4.0 on xenial and bionic, else install Mongo 3.6
```

Comment:
There is no `else` anymore since we switch to Mongo `4.0` for all the platforms. That's the point.

Keep in mind that installation for the `stable` and older st2 versions are handled via branches like https://github.com/StackStorm/st2-packages/tree/v3.2

Reply:
I see your point. I was thinking something else.